ARC specification.md

```diff
@@ -252,10 +252,16 @@ The file `arc.cwl` SHOULD exist at the root directory of each ARC. It describes
 
 ## Data Path Annotation
 
-All metadata references to files or directories located inside the ARC MUST follow the following patterns:
+All metadata references to files or directories MUST follow one of the following patterns:
 
 - The `general pattern`, which is universally applicable and SHOULD be used to specify the path relative to the ARC root.
 
+- The `URL pattern`, which is universally applicable and MAY be used to specify path locations anywhere on the web, given the following constraints:
```

Comment:
SHOULD instead of MAY. Maybe even MUST, if we want to say that this pattern must be used for urls. In this case we also need to update to MUST for the general case.

Reply:
Hey, thanks for your feedback. 

Actually I wanted to describe it from another angle. Kind of like:

- You SHOULD put the files in the ARC and use the general pattern
- You MAY keep files outside and use the URL pattern

But that might actually be too obtrusive, and also doesn't become clear. So I'll look into it.